docker-initunlocklnd.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+echo "[lnd_unlock] Waiting 2 seconds for lnd..."
+sleep 2;
```

Comment:
Don't do this, this will be fragile. Instead make sure it graciously handle lnd being down.

Reply:
Implemented